Gems/AudioEngineWwise/Code/Source/Editor/AudioWwiseLoader.cpp

```diff
@@ -79,15 +72,15 @@ namespace AudioControls
                     // we load only one as all of them should have the
                     // same content (in the future we want to have a
                     // consistency report to highlight if this is not the case)
-                    m_localizationFolder = fileName;
+                    m_localizationFolder.assign(fileName.Native().data(), fileName.Native().size());
                     LoadSoundBanks(rootFolder, m_localizationFolder, true);
                     isLocalizedLoaded = true;
                 }
             }
-            else if (AZ::StringFunc::Find(fileName.c_str(), Audio::Wwise::BankExtension) != AZStd::string::npos
-                && !AZ::StringFunc::Equal(fileName.c_str(), Audio::Wwise::InitBank))
+            else if (fileName.Extension() == Audio::Wwise::BankExtension && !AZ::StringFunc::Equal(fileName.Native(), Audio::Wwise::InitBank))
```

Comment:
This should just use the `AZ::IO::PathView::operator==` comparison function.
It performs a case insensitive compare of the path on Windows
```suggestion
            else if (fileName.Extension() == Audio::Wwise::BankExtension && fileName != Audio::Wwise::InitBank)
```

Reply:
Nice, I think it was case-insensitivity that I was depending on.